### PR TITLE
Use let or const instead of var

### PR DESCRIPTION
**Pull request by czosel**

### CODE REVIEW

File: src/parser/expr.js [hunk 1]
```diff
@@ -204,28 +204,28 @@ module.exports = {
       case this.tok.T_NEW:
         return this.next().read_new_expr();
 
-      case this.tok.T_ISSET:
+      case this.tok.T_ISSET: {
```
Comment (ichiriac) [inline src/parser/expr.js]: why surrounding with `{` ... `}` ? It it a linter directive or is it better ?

File: src/ast/variadic.js [hunk 2]
```diff
@@ -4,19 +4,19 @@
...
 /**
  * Introduce a list of items into the arguments of the call
- * @constructor Variadic
+ * @constructor variadic
  * @extends {Expression}
  * @property {Array|Expression} what
  * @see https://wiki.php.net/rfc/argument_unpacking
  */
-var Variadic = Expr.extends(function Variadic(what, location) {
+const variadic = Expr.extends(function variadic(what, location) {
```
[the remark this answers is not in the document]
Comment (czosel) [reply src/ast/variadic.js]: of course, this was a search/replace mistake :wink:

File: src/parser/expr.js (hunk 1 above)
Comment (czosel) [reply src/parser/expr.js]: Yeah, eslint complains when you put `let`/`const` inside `case`. see https://eslint.org/docs/rules/no-case-declarations

**Comment by czosel** [comment]
:smile: Thanks for the invite!

**Comment by ichiriac** [comment]
You are very welcome my friend, I'm glad for your help :+1:

**Pull request merged**